GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1263,6 +1263,24 @@ private void FillCommitInfo(GitRevision? revision)
 
         private async Task FillGpgInfoAsync(GitRevision? revision)
         {
+            // Don't show the "GPG" tab for artificial commits
+            bool showGpgInfoTab = revision?.IsArtificial != true && AppSettings.ShowGpgInformation.Value;
+
+            if (showGpgInfoTab)
+            {
+                if (GpgInfoTabPage.Parent is null)
+                {
+                    int index = CommitInfoTabControl.TabPages.IndexOf(TreeTabPage);
+                    Debug.Assert(index != -1, "TabControl should contain file tree tab page");
```

Comment:
Use DebugHelper

Reply:
I did the fix on master and that's what I used but I think it's not available in release branch...

Reply:
Ah, makes sense. Thank you.